release-notes/Cloud/dataminer_services_change_log_2025.md

```diff
@@ -9,6 +9,10 @@ The dataminer.services platform gets updated continuously. This change log can h
 > [!NOTE]
 > Many features on dataminer.services are dependent on DxMs. You can find the change logs for these under [DxM release notes](xref:DxM_RNs_index).
 
+### TDB - Enhancement - Admin - Unmanaged Objects Usage visibility and export posibility
+
+In the Admin app, the usage for unmanaged objects will now be displayed if you are using at least DataMiner 10.5.5 and DataMiner SupportAssistant 1.7.3. The data will also be available for export.
```

Comment:
Do you also know what the minimum main release version is? We try to always mention both, so the minimum version is clear regardless of the upgrade track the user is on.